### PR TITLE
Vote card

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by Caroline-CD — APPROVED** [review]
OK :+1:

File: src/Components/VoteCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// import LinearProgress from '@mui/material/LinearProgress';
```
Comment (Hacene010) [inline src/Components/VoteCard/index.jsx]: dead code !!!

File: src/Components/VoteCard/style.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+  progress {
+    background-color: var(--color--primary);
+    border: 0;
+    width: 100%;
+    height: 40px;
+    border-radius: 10px;
+    border: none;
+  }
+  progress::-webkit-progress-bar {
```
Comment (Hacene010) [inline src/Components/VoteCard/style.jsx:64]: if you add a blank line before, why don't you add it here?

File: src/Components/VoteCard/index.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    } else setVoteValue(voteValue + 1);
+  }
+
+  return (
+    <>
+      <VoteContainer>
+        <img src="https://i.imgur.com/XW6WVVR.png" alt="VW Golf 1 GTI" />
+        <h1 className="titleArea">VW Golf 1 GTI</h1>
+        <p className="PriceArea">A partir de 30 000€</p>
+        <button type="button" onClick={upvote}>
```
Comment (Hacene010) [inline src/Components/VoteCard/index.jsx]: camelCase into the onClick

File: src/Pages/VotePage/index.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        <div className="TopVoteCardWrapper">
+          <VoteCard />
+          <VoteCard />
+          <VoteCard />
+        </div>
+      </div>
+
+      <div className="h3Container">
+        <h3>
+          Votez pour le véhicule que vous souhaitez rétrofité. Une fois le
```
Comment (Hacene010) [inline src/Pages/VotePage/index.jsx]: a small content change "...que vous souhaitez rétrofiter.."

File: src/Pages/VotePage/style.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+
+const VoteContainer = styled.div`
+  margin: 5% 5%;
+  height: 100%;
+  padding: 15px;
+
+  display: flex;
+  flex-direction: column;
+
```
Comment (Hacene010) [inline src/Pages/VotePage/style.jsx]: empty line

File: src/Pages/VotePage/style.jsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+
+const VoteContainer = styled.div`
+  margin: 5% 5%;
+  height: 100%;
+  padding: 15px;
+
```
Comment (Hacene010) [inline src/Pages/VotePage/style.jsx]: empty line

File: src/Pages/VotePage/style.jsx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  div.VoteCardContainer {
+    margin: 0% 5%;
+    display: flex;
+    flex-direction: row;
```
Comment (Hacene010) [inline src/Pages/VotePage/style.jsx]: You can use flex-flow: row wrap; here.

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some evils to correct

**Review by Caroline-CD — DISMISSED** [review]
Ok :+1: for me

**Review by Morel-Victor — DISMISSED** [review]
Ok for me !

**Review by Hacene010 — DISMISSED** [review]

**Review by Morel-Victor — DISMISSED** [review]

**Review by Morel-Victor — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**